### DISCUSSION TITLE
Proxmox LXC Updater error with --ostype unmanaged

### DISCUSSION THREAD

**Discussion by ednancosta** [tteck/Proxmox#962] [Help Needed] [answered]
When using the scrip "Proxmox LXC Updater", as working good and whell.
Until i install OpenWRT in a ostype unmanaged, fallowing this tutorial: https://virtualizeeverything.com/2022/05/23/setting-openwrt-in-proxmox-lxc/

Proxmox VM is the lowest VM number.
When the script is runing return the error: 
`lxc-attach: 101: ../src/lxc/attach.c: lxc_attach_run_command: 1839 No such file or directory - Failed to exec "hostname"`

And exit the script.

![image](https://user-images.githubusercontent.com/83416189/213841778-25069b21-479e-4b5b-86f3-e5cc9921b970.png)

====
Maybe some way to skip this (without turning off, because as the router, cant turn off).

**Comment by tteck** [comment, marked as answer]
Looks like your left with updating each LXC individually.

**Reply by ednancosta** [reply]
No way to script skip the CT having problem?
Or skip with a argument? (--skip 101)
